Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverHandler.cs

```diff
@@ -88,6 +113,43 @@ private void LateUpdate()
             }
         }
 
+        public virtual void AttachToNewTrackedObject()
+        {
+
+            switch (trackedObjectToReference)
+            {
+                case TrackedObjectToReferenceEnum.MotionControllerLeft:
+                default:
+                    Handedness = UnityEngine.XR.WSA.Input.InteractionSourceHandedness.Left;
+                    break;
+                case TrackedObjectToReferenceEnum.MotionControllerRight:
+                    Handedness = UnityEngine.XR.WSA.Input.InteractionSourceHandedness.Right;
+                    break;
+            }
+
+            // We need to reattach to the appropriate controller
+            TryAndAddControllerTransform();
+            foreach (Solver solver in m_Solvers)
+            {
+                solver.SeekTrackedObject();
+            }
+        }
+
+        public bool RequiresOffset()
```

Comment:
could this be a property instead?

Reply:
It should be private, except Solver needs it for the coroutine that should be in SolverHandler. I added a TODO for making it that way with Kurtis's refactor.